.github/workflows/ci.yml

```
@@ -156,11 +157,38 @@ jobs:
             --eval "(push \"$(pwd)/lisp\" load-path)" \
             -f batch-byte-compile lisp/*.el
 
-      - name: Check package metadata
+      - name: Package lint
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC1009:info:6:10: The mentioned syntax error was in this double quoted string

(shellcheck)